### PR TITLE
Feature/DEI-247 update description axis_filter_rule

### PR DESCRIPTION
**Pull request by CindyvdVries**

### CODE REVIEW

File: docs/manual/rules/axis_filter_rule.md [hunk 1]
```diff
@@ -5,11 +5,10 @@ FORMAT
 - axis_filter_rule:
       name: <name_of_rule_in_text>
       description: <description_of_rule_in_text>
-      layer_number: <integer_nr_of_layer>
-	  axis_name: <name_of_axis_applied>
+      axis_name: <name_of_axis_applied>
       layer_number: <integer_nr_of_layer_in_axis_applied>
       input_variable: <one_3D_input_variable_name>
-      output_variable: <one_output_variable_name>
+      output_variable: <one_output_variable_name>
```
Comment (mKlapwijk) [inline docs/manual/rules/axis_filter_rule.md:11]: why add a trailing space?

File: template_input.yaml [hunk 2]
```diff
@@ -54,13 +54,40 @@ rules:
...
         output_variable: 
+ 
+  - formula_rule:
+      name: 
+      description:
+      formula:
+      input_variables: [,]
+      output_variable:
 
-  - combine_results_rule:
```
Comment (mKlapwijk) [inline template_input.yaml:58]: why is this removed?

**Review by mKlapwijk — CHANGES_REQUESTED** [review]

File: template_input.yaml (hunk 2 above)
Comment (CindyvdVries) [reply template_input.yaml:58]: they thought this rule was so nice, they added it twice ;)

File: docs/manual/rules/axis_filter_rule.md (hunk 1 above)
Comment (CindyvdVries) [reply docs/manual/rules/axis_filter_rule.md:11]: if you open the file, it is a sea of trailing spaces :) updated it. No more trail of spaces.

File: template_input.yaml (hunk 2 above)
Comment (mKlapwijk) [reply template_input.yaml:58]: :)

**Review by mKlapwijk — APPROVED** [review]
